lib/super_digit.rb

```diff
@@ -1,11 +1,38 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(logn)
+# We are dividing by 10 each time so I think it is O(logn)
+# Space Complexity - O(n)
+# The stack grows in proportion to the size of n
+
 def super_digit(n)
```

Comment:
Since you're doing the sum O(d) inside the recursion which goes d times, where d is the number of digits this is O( (log n)^2)